clock_exercise/TimerUpDownNumberField.qml

```diff
@@ -0,0 +1,69 @@
+import QtQuick 2.0
+
+Item {
+    id: root
+
+    property alias text: textField.text
+    property int value: 0
+    property int max: 99
+
+    y: 160
+    width: 190
+    height: 502
+
+    Rectangle {
+        width: 190
+        height: 280
+        color: "#1B2F46"
+        radius: 30
+
+        Text {
+            x: 0
+            y: 43
+            width: 190
+            height: 116
+            text: root.value < 10? "0" + root.value.toString(): root.value.toString()
+            color: "#FCB647"
+            font.pixelSize: 120
+            horizontalAlignment: Text.AlignHCenter
+            font.family: "Buenos Aires"
+            font.styleName: "Thin"
+        }
+
+        Text {
+            id: textField
+
+            x: 73
+            y: 215
+            width: 44
+            height: 24
+            color: "#9FAAB0"
+            font.pixelSize: 20
+            horizontalAlignment: Text.AlignHCenter
+            font.family: "Buenos Aires"
+            font.styleName: "Regular"
+        }
+    }
+
+    ArrowButton {
+        id: upButton
+
+        y: 290
+        onClicked: root.value = Math.min(root.value + 1, root.max)
```

Comment:
Che ne pensi se prevedessimo il wrap-around quando si arriva in fondo?

Ogni volta che c'è un assegnamento io racchiudo sempre l'handler di un evento tra parentesi graffe, gli unici casi in cui non uso parentesi graffe sono quelli in cui chiamo semplicemente un segnale o una funzione (es. `onClicked: root.back()`). E' possibile che la cosa mi abbia dato problemi in passato, ma non ricordo il caso in questo momento...

Reply:
si il wrap-around è più corretto